cmd/kubeadm/app/cmd/util/join.go

```diff
@@ -81,6 +85,9 @@ func GetJoinCommand(kubeConfigFile, token, key string, skipTokenPrint, uploadCer
 	if skipTokenPrint {
 		ctx["Token"] = template.HTML("<value withheld>")
 	}
+	if skipCertificateKeyPrint {
+		ctx["CertificateKey"] = template.HTML("<value withheld>")
```

Comment:
Oddly enough, we use `html/template` here. This might cause problems when we try to feed a template with non-alphanumeric characters (`<` is turned into `&lt;`, etc.).
But anyway, this again is out of the scope of this PR.